schematics/src/utils/json-utils.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * set the value for the path provided in keys.
+ *
+ * @param object Object like a json, for example
+ * @param keys keys to access the property (string like key1.key2 or ['key1', 'key2'])
+ * @param val value to set at this place
+ */
+export function setProp(object, keys, val) {
+  keys = Array.isArray(keys) ? keys : keys.split('.');
+  if (keys.length > 1) {
+    object[keys[0]] = object[keys[0]] || {};
+    return setProp(object[keys[0]], keys.slice(1), val);
+  }
+  object[keys[0]] = val;
+}
+
+/**
+ * This function works like the 'get' as Lodash
+ *
+ * @param object Object like a json, for example.
+ * @param path Generally a string like key1.key2 or array like ['key1', 'key2']
+ * @param defaultValue if a value is not found, provide a default value. default: undefined
+ */
+export function getProp(object, path, defaultValue = undefined) {
```

Comment:
I don't see that you're using this function is it redundant? if so, please remove it

Reply:
It was there for future improvement, but I will remove it